src/psyclone/psyir/symbols/symbol_table.py

```diff
@@ -1249,6 +1261,11 @@ def resolve_imports(self, container_symbols=None, symbol_target=None):
                         new_symbol.visibility = self.default_visibility
                         self.add(new_symbol)
 
+                if symbol_target:
```

Comment:
Why has this been moved? If there was a problem with where it was before please could you add a test to show that the issue has been fixed.

Reply:
I reverted this change since I couldn't trigger the second problem Simon was reporting (maybe the capitalization also fixed it?). Anyway, I think the PR can continue without it because it fixes a known problem and if Simon still sees an issue I will ask for a more concrete example of it.